CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.5.0-beta.1 (TBD)
```

Comment:
This should be `1.4.0-beta.2` since `1.4.0` hasn't been released yet. Then we can roll up all of the new features into `v1.4.0`.

Reply:
Oh that would be a timesaver!